### ISSUE TITLE
Listing of Vizzes you've Upvoted

### ISSUE THREAD

**Issue by curran** [datavis-tech/vizhub-issue-tracker#611]
As a viz viewer who has upvoted some vizzes in the past, I want to be able to see a listing of all the vizzes I've upvoted.

**Comment by nitanagdeote** [comment]
I don't think the upvote feature is that popular in VizHub yet.

**Comment by curran** [comment]
Indeed, it's not really a complete feature set yet around upvoting.

The main point of upvoting, in the future vision for it, is to inform the ordering on the home page.

I'd like to adapt something like [Reddit's best algorithm](https://github.com/clux/decay) for sorting the home page.

Here are the upvote-related issues:

 * Introduce upvotesCount field #361
 * Trending algorithm from upvotes for home page #478
 * Listing of Vizzes you've Upvoted #611
 * Feature suggestion: Star your viz #610
 * "Upvote Icon" change to gold star #566
 * Page for listing people who upvoted your viz. #467
 * On mobile 'upvote' icon #523
 * Upvote from mobile #429
